inference_lib/setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = aqlm
+version = 1.0.0dev9
+author = Vage Egiazarian, Andrei Panferov and Elias Frantar
```

Comment:
It is your decision. but I would recommend "AQLM paper authors" __if you have no preference__

If you chose this deliberately, I have no objections.

Reply:
"AQLM paper authors" makes perfect sense. Will do that